README.md

```diff
@@ -11,8 +11,26 @@ organisation of neuroimaging data.
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
-Open or comment on one of our [NeuroStars issues](https://neurostars.org/tags/bids) or check out the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit)!
+# Using BIDS
+
+To use BIDS, all you need is neuro data, a computer, and the
```

Comment:
I couldn't find a list of the available modalities in the specification itself, just as links under "modality specific files." So, for now, this link goes to the GitHub folder. Better suggestions would be appreciated !

```suggestion
To use BIDS, all you need is neuroimaging data (see [supported modalities here](https://github.com/bids-standard/bids-specification/tree/master/src/04-modality-specific-files)), a computer, and the
```

Reply:
I think it's a good idea to add supported modalities! Perhaps it'd be good for now to simply write them down?

Reply:
That does seem like the simplest path forward, for now :) 

We'll just need to remember to keep it sync'ed moving forward !